### PR TITLE
ocp-edge qe: add puller account

### PR DESCRIPTION
**Pull request by yprokule**

### CODE REVIEW

File: services/edge/admin_rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+metadata:
+  name: ocs-image-puller-binding
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: ocp
```
Comment (stevekuznetsov) [inline services/edge/admin_rbac.yaml:26]: This should be `edge`

Comment (yprokule) [reply services/edge/admin_rbac.yaml]: Updated, also has to be updated in `services/ocs/admin_rbac.yaml` ?

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**